stories/Select/select.guideline.mdx

```diff
@@ -0,0 +1,73 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  Primary,
+  Controls,
+  Unstyled,
+  Source,
+} from '@storybook/blocks';
+import { Select } from '../../src/lib/components/selectv2/Selectv2.component';
+
+import * as SelectStories from './selectv2.stories';
+
+<Meta title="Guideline" of={SelectStories} />
+
+# Select
```

Comment:
@Cuervino maybe for later on we need to think about a loading state for select component requiring async process to get the options